examples/static-site-gen/platform/src/highlight.rs

```diff
@@ -26,18 +26,15 @@ pub fn highlight_roc_code(code: &str) -> String {
             Token::Number => {
                 buf = push_html_span(buf, current_text, "number");
             }
-            Token::Pipe => {
+            Token::Pizza => {
                 buf = push_html_span(buf, current_text, "pipe");
             }
-            Token::Arrow => {
+            Token::Arrow | Token::Backpass => {
                 buf = push_html_span(buf, current_text, "arrow");
             }
             Token::Bar => {
                 buf = push_html_span(buf, current_text, "bar");
             }
-            Token::Backpass => {
-                buf = push_html_span(buf, current_text, "backpass");
-            }
```

Comment:
is this by design? I guess they are both arrows

Reply:
I was splitting everything out into separate classes, but there are a lot of tokens and so this is an attempt to limit the number by grouping loosely.